x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v7_14_0.test.ts

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectMigrationContext, SavedObjectUnsanitizedDoc } from 'kibana/server';
+
+import type { PackagePolicy } from '../../../../common';
+
+import { migrateEndpointPackagePolicyToV7140 } from './to_v7_14_0';
+
+describe('7.14.0 Endpoint Package Policy migration', () => {
+  const migration = migrateEndpointPackagePolicyToV7140;
+  it('adds supported option for ransomware on migrations', () => {
+    const doc = {
```

Comment:
Any chance you can collapse several of these document mocks into a builder function instead?

Reply:
@paul-tavares my next PR is Linux Malware and will build on this migration test, I'll address this there.